packages/wrangler/src/api/startDevWorker/ProxyController.ts

```diff
@@ -67,7 +67,8 @@ export class ProxyController extends EventEmitter {
 			workers: [
 				{
 					name: "ProxyWorker",
-					compatibilityFlags: ["nodejs_compat"],
+					// `url_standard` required to parse IPv6 hostnames correctly
+					compatibilityFlags: ["nodejs_compat", "url_standard"],
```

Comment:
`url_standard` is default as of over a year ago—can we updated the compatibility date for this worker?